bin/run_faucet

```diff
@@ -106,6 +108,19 @@ cid=$(docker run -d --privileged --name $CONTAINER \
              $docker_volumes $docker_envs \
              -p $ext_ofpt:6653 $port_opts $IMAGE)
 
+if [ $IMAGE == "forch/faucet" ]; then
+    pid=$(docker inspect --format="{{ .State.Pid }}" $cid)
+    data_intf_name=data_fauc_$faucet_id
+    sudo ip link add $data_intf_name type veth peer name data0 netns $pid
+    sudo ip link set $data_intf_name up
+    if [[ $faucet_id =~ ^[0-9]+$ ]] ; then
+        docker exec $CONTAINER ip link add link data0 name data0.272 type vlan id 272
+        docker exec $CONTAINER ip addr add 192.168.1.1$faucet_id/24 dev data0.272
```

Comment:
this would be much better served as 192.168.2.$faucet_id

Reply:
Done